community-operators/metering/meteringoperator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -57,15 +166,23 @@ spec:
     * **Configure AWS Billing Data Source** - To assign Pod $$ costs on AWS, create a [read-only IAM user](https://github.com/operator-framework/operator-metering/blob/master/Documentation/configuring-aws-billing.md) ([example-policy](https://github.com/operator-framework/operator-metering/blob/master/Documentation/aws/read-only.json)) and [configure Metering](https://github.com/operator-framework/operator-metering/blob/master/manifests/metering-config/aws-billing.yaml) to use it.
 
   keywords: [metering metrics reporting prometheus chargeback]
```

Comment:
Please add commas to this list, or use a yaml list for the keywords

Reply:
Why? It's not necessary. 

Also, most of this is autogenerated from other tools so the yaml it outputs is whatever the encoder determines to be the simplest yaml it can output.

Reply:
To clarify, we even do specify it this way in our config files: https://github.com/operator-framework/operator-metering/blob/master/manifests/deploy/common-olm-values.yaml#L34

But it outputs it with the minimum/"simplest" required syntax generally: https://github.com/operator-framework/operator-metering/blob/master/manifests/deploy/openshift/olm/bundle/meteringoperator.v4.1.0.clusterserviceversion.yaml#L168